pkg/container/docker/squid.go

```diff
@@ -207,6 +196,22 @@ func createTempSquidConf(
 	return tmpFile.Name(), nil
 }
 
+func writeCommonConfig(sb *strings.Builder, serverHostname string) {
```

Comment:
i like your idea, but can we repurpose in a different way? make "writeCommonConfig" reusable as well, but accept a bool parameter for "expose_port". When set, write the http_port string and ignore if not. So "writeCommonConfig(&sb, serverHostname+"-ingress", false)"
"writeCommonConfig(&sb, serverHostname+"-egress", true)"